SasquatchMac/SasquatchMac/ViewControllers/AppCenterViewController.swift

```diff
@@ -133,4 +134,35 @@ class AppCenterViewController : NSViewController, NSTextFieldDelegate {
     }
   }
 
+  @IBAction func setLogURL(_ sender: NSButton) {
+    let alert: NSAlert = NSAlert()
+    alert.messageText = "Log URL"
+    alert.addButton(withTitle: "Reset")
+    alert.addButton(withTitle: "Save")
+    alert.addButton(withTitle: "Cancel")
+    let scrollView: NSScrollView = NSScrollView(frame: NSRect(x: 0, y: 0, width: 300, height: 40))
+    let textView: NSTextView = NSTextView(frame: NSRect(x: 0, y: 0, width: 290, height: 40))
+    let defaultText = "Log URL currently set to the production"
+    textView.string = UserDefaults.standard.string(forKey: kMSLogUrl) ?? defaultText
+    scrollView.documentView = textView
+    scrollView.hasVerticalScroller = true
+    scrollView.contentView.scroll(NSPoint(x: 0, y: textView.frame.size.height))
+    alert.accessoryView = scrollView
+    alert.alertStyle = NSWarningAlertStyle
+    switch(alert.runModal()) {
+    case NSAlertFirstButtonReturn:
+      UserDefaults.standard.set(nil, forKey: kMSLogUrl)
```

Comment:
It should also call the setLogUrl right after persist the URL in UserDefaults. Missing calling code.

Reply:
This feature need restart app to work, when restart app, it will call setLogUrl before AppCenter start in AppDetegate. So I think it don't need call setLogUrl after press button.

Reply:
Can app refresh the log URL‘s value immediately after changing it？

Reply:
Good point @jinquan2019. We have two scenarios to test `setLogUrl` that is called before `start` and after `start`. Calling `setLogUrl` when an endpoint is changed will cover after `start` scenario.

Reply:
Thanks for above point, I missed the scenario: Calling `setLogUrl `when an endpoint is changed will cover after `start`. Added it in the new commit.